source/PlayerInfo.cpp

```diff
@@ -337,6 +337,9 @@ void PlayerInfo::Load(const string &path)
 			availableMissions.emplace_back(child);
 		else if(child.Token(0) == "conditions")
 			conditions.Load(child);
+		else if(child.Token(0) == "raid attraction")
+			for(const DataNode &grand : child)
+				raidAttraction[GameData::Governments().Find(grand.Token(0))] = grand.Value(1);
```

Comment:
what if find returns a nullptr?

Reply:
I think I saw this logic used somewhere else, but thats a good point

Reply:
You should be using Get during loads.

Reply:
```suggestion
				raidAttraction[GameData::Governments().Get(grand.Token(0))] = grand.Value(1);
```

Reply:
I looked through all Find calls, this is not happening elsewhere